.github/pull_request_template.md

```diff
@@ -3,11 +3,12 @@ Related Ticket:
 Description of Changes:
 * Placeholder text
 
-DOD:
-- [ ] nav.yml has been updated if applicable
-- [ ] file has been included where required if applicable
-- [ ] files removed have been deleted, not just excluded from the build if applicable
+Pre-checks:
+- [ ] Branch prefixed with `feature/` or `hotfix/`
+- [ ] `_data/nav.yml` has been updated (if applicable)
+- [ ] file has been included where required (if applicable)
```

Comment:
file -> files?

Reply:
Also the capitalisation of the different lines is inconsistent?

Reply:
Thanks. Fixed in 790cf223cb47e77b1ee9f0bcf53f4ee00f8c5a61